app/components/post_list/scroll_to_end_view.tsx

```diff
@@ -97,7 +97,7 @@ const ScrollToEndView = ({
         () => ({
             transform: [
                 {
-                    translateY: withTiming(showScrollToEndBtn ? -80 - keyboardOverlap - bottomAdjustment : 0, {duration: 300}),
+                    translateY: withTiming(showScrollToEndBtn ? -100 - keyboardOverlap - bottomAdjustment : 0, {duration: 300}),
```

Comment:
Change delta of "-20" has not much of a significant, but it would be nice to know why it was made.  Diff in screenshot would help, but I'm happy with quick explanation.

Reply:
This is how it looks with -80

<img width="441" alt="Screenshot 2025-02-03 at 12 36 16 PM" src="https://github.com/user-attachments/assets/84e16f84-282e-439d-8686-703397c78b53" />

This is how it looks with -100

<img width="443" alt="Screenshot 2025-02-03 at 12 49 08 PM" src="https://github.com/user-attachments/assets/abc90413-af14-4da5-a7ef-b94df0985517" />

adjustment of 20 is all we need. Maybe a few pixels here or there more, but thats to be done later during a UX review.